### DISCUSSION TITLE
My Account is Suspended.

### DISCUSSION THREAD

**Discussion by parth-coderscotch** [community/community#110586] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hi
My GitHub account is suspended.
I didn't received any mail regarding this, when i try to login it show "Account suspended"
![Screenshot_1](https://github.com/community/community/assets/75311603/5a658f16-406d-4f58-8190-6aa8e93ac670)

I sent an email to support@githubsupport.com, its been 5-6 days but they are not responding to mail

Is there any other way to unblock account.

My username is: parth35

**Comment by git-sumana** [comment]
Response #1
If your GitHub account has been suspended without receiving any notification emails, there are several steps you can take to address the issue:

1. Check Spam or Junk Folders: 
Sometimes, notification emails may be mistakenly filtered into the spam or junk folder of your email inbox. Check these folders to see if any emails from GitHub are present.

**Comment by git-sumana** [comment]
Response #2
Contact GitHub Support: If you cannot find any notification emails and are unsure why your account was suspended, you should reach out to GitHub support for assistance. You can submit a request to GitHub's support team explaining your situation and asking for clarification on the reason for the suspension. Provide any relevant details, such as your username and any recent activities on your account, to help them investigate the issue.

**Comment by git-sumana** [comment]
Response #3
Review GitHub's terms of service and community guidelines to ensure that you have not violated any of their policies. If you believe your account was suspended in error or due to a misunderstanding, you can explain your case to GitHub support.

**Comment by git-sumana** [comment]
Response #4
Appeal the Suspension: If you believe your account was suspended unfairly, you can appeal the suspension by following the instructions provided in GitHub's suspension notification or by contacting GitHub support directly. Provide any evidence or information that supports your case and demonstrates that your account should be reinstated.

**Comment by git-sumana** [comment]
Response #5
Wait for a Response: After reaching out to GitHub support or submitting an appeal, you may need to wait for a response while they investigate the issue. Be patient and check your email regularly for updates from GitHub.

**Comment by git-sumana** [comment]
#Response 6
Consider Alternative Communication Channels: If you have not received a response from GitHub support within a reasonable timeframe, you may consider reaching out to them through alternative communication channels, such as their official Twitter account or community forums.

Remember to remain courteous and provide all relevant information when communicating with GitHub support. They are there to assist you and resolve any issues you may encounter with your account.

**Comment by git-sumana** [comment]
Accept the comments so that it would be even useful to other, i know the pain of loosing a git acc

**Reply by queenofcorgis** [reply]
@git-sumana, 
We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @parth-coderscotch, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
